spec/spec_helper.rb

```diff
@@ -1,19 +1,29 @@
 # frozen_string_literal: true
 
-require "byebug"
-require "simplecov"
-
 ENV["RAILS_ENV"] ||= "test"
 require File.expand_path("../../config/environment", __FILE__)
 require "rspec/rails"
 
+require "byebug"
+require "govuk-content-schema-test-helpers/rspec_matchers"
```

Comment:
We should use `govuk_schemas` for this (it's already in the Gemfile). It exposes the same API but doesn't require the setup.

This is all that's needed:

```rb
require 'govuk_schemas/rspec_matchers'
```